src/components/UserProfile/InterestsAutocomplete.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { debounce, get } from 'lodash';
+import { compose, withProps, withPropsOnChange, withState } from 'recompose';
+import { css } from 'react-emotion';
+import { withTheme } from 'emotion-theming';
+import Downshift from 'downshift';
+import { Trans } from 'react-i18next';
+
+import { withApi } from 'services/api';
+import { getTags } from 'services/profiles';
+
+const DropdownLabel = compose(withTheme)(({ theme, children }) => (
+  <span
+    className={css`
+      text-transform: uppercase;
```

Comment:
I think we should start trying to move new styles outside of components, maybe just in a local style js file so it's easier to move things around later :)